test/e2e/specs/editor/various/popovers.spec.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor state', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.beforeEach( async ( { pageUtils } ) => {
+		await pageUtils.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.describe( 'dropdown', () => {
+		test( 'toggles via click', async ( { page } ) => {
+			const isMoreMenuOpen = async () =>
+				!! ( await page.locator(
+					'.interface-more-menu-dropdown__content'
+				) );
```

Comment:
`page.locator` will always return a truthy result and will not return a promise. You can see more information about locators in the [docs](https://playwright.dev/docs/locators).

Moreover, as per suggested in the [best practices](https://playwright.dev/docs/locators), we should prioritize accessible selectors over class name selectors.

In this case I think a selector like `role=menu[name="Options"i]` is more suitable.